### PR TITLE
Mark basic JS features as supported in Fx 1 and FxA 4

### PR DESCRIPTION
**Pull request by Elchi3**
I was bored in the train from Brussels to Bremen. All these JS features are pretty old and should be in Firefox 1 (and Fx Android 4).

### CODE REVIEW

File: javascript/builtins/Function.json [hunk 1]
```diff
@@ -122,10 +122,10 @@
                 "version_added": false
               },
               "firefox": {
-                "version_added": false
+                "version_added": "1"
```
Comment (teoli2003) [inline javascript/builtins/Function.json]: This looks weird; can you confirm it is correct (I don't think so, at least not without a version_removed I guess)

File: javascript/builtins/Function.json [hunk 2]
```diff
@@ -122,10 +122,10 @@
...
               },
               "firefox_android": {
-                "version_added": false
+                "version_added": "4"
```
Comment (teoli2003) [inline javascript/builtins/Function.json]: This looks weird; can you confirm it is correct (I don't think so, at least not without a version_removed I guess)

File: javascript/statements.json [hunk 3]
```diff
@@ -1008,11 +1008,11 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "13",
+              "version_added": "14",
```
Comment (teoli2003) [inline javascript/statements.json]: Firefox 13 for developers still has it listed! (And not Firefox 14 for developers). Please change both pages if this change was indeed correct

I wasn't able to find the bug to decide if 13 or 14 is correct.

File: javascript/statements.json [hunk 4]
```diff
@@ -1008,11 +1008,11 @@
...
               "notes": "Prior to Firefox 51, using the <code>for...of</code> loop construct with the <code>const</code> keyword threw a <code>SyntaxError</code> (\"missing = in const declaration\")."
             },
             "firefox_android": {
-              "version_added": true
+              "version_added": "14"
```
Comment (teoli2003) [inline javascript/statements.json]: Same as above + the note is likely to be copied too.

File: javascript/builtins/Array.json [hunk 5]
```diff
@@ -836,10 +836,10 @@
                 "version_added": true
               },
               "firefox": {
-                "version_added": true
+                "version_added": "1.5"
```
Comment (teoli2003) [inline javascript/builtins/Array.json]: Likely correct, but can you double check that you indeed wanted 1.5 there. It is not listed in http://www.squarefree.com/burningedge/releases/1.5-comprehensive.html .

Comment (Elchi3) [reply javascript/builtins/Array.json]: This is like `indexOf` and see also https://developer.mozilla.org/en-US/docs/Web/JavaScript/New_in_JavaScript/1.6

File: javascript/builtins/Function.json (hunk 1 above)
Comment (Elchi3) [reply javascript/builtins/Function.json]: Good catch, I got this wrong. Back to false.

File: javascript/builtins/Function.json (hunk 2 above)
Comment (Elchi3) [reply javascript/builtins/Function.json]: Yep, same here, changed it back to false.

File: javascript/statements.json (hunk 3 above)
Comment (Elchi3) [reply javascript/statements.json]: 13 is correct, I wanted to add 14 for Android as 13 doesn't exist for Android but accidentally also changed the Firefox desktop version. Good catch!

File: javascript/statements.json (hunk 4 above)
Comment (Elchi3) [reply javascript/statements.json]: I copied the note.

**Pull request merged**